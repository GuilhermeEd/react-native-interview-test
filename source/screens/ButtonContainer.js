import React, { Component } from 'react';
import { View } from 'react-native';
import styles from './styles';

export default class ButtonContainer extends Component {
  render() {
    return (
      <View className='buttonContainer' style={styles.buttonContainer}>
        {this.props.children}
      </View>
    )
  }
}
