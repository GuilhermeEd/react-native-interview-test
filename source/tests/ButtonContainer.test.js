import React from 'react';
import renderer from 'react-test-renderer';
import ButtonContainer from "../screens/ButtonContainer";

test('<ButtonContainer> renders correctly', () => {
    const tree = renderer.create(<ButtonContainer />).toJSON();
    expect(tree).toMatchSnapshot();
});