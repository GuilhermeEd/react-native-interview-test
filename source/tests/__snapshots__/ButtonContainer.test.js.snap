// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonContainer> renders correctly 1`] = `
<View
  className="buttonContainer"
  style={
    Object {
      "display": "flex",
      "flex": 1,
      "flexDirection": "row",
      "height": 60,
      "justifyContent": "space-around",
    }
  }
/>
`;
